specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -502,7 +502,429 @@
+            "description": "Hazy Moonlight"
+          },
+          {
+            "value": 38,
+            "name": "Mostly Cloudy",
+            "description": "Mostly Cloudy"
+          },
+          {
+            "value": 39,
+            "name": "Partly Cloudy with Showers",
+            "description": "Partly Cloudy with Showers"
+          },
+          {
+            "value": 40,
+            "name": "Mostly Clear",
+            "description": "Mostly Clear"
+          },
+          {
+            "value": 41,
+            "name": "Partly Cloudy with Thunderstorms",
+            "description": "Partly Cloudy with Thunderstorms"
+          },
+          {
+            "value": 42,
+            "name": "Mostly Cloudy with Thunderstorms",
+            "description": "Mostly Cloudy with Thunderstorms"
+          },
+          {
+            "value": 43,
+            "name": "Mostly Cloudy with Flurries",
+            "description": "Mostly Cloudy with Flurries"
+          },
+          {
+            "value": 44,
+            "name": "Mostly Cloudy with Snow",
+            "description": "Mostly Cloudy with Snow"
+          }
+        ]
+      }
+    },
+    "UnitType": {
+      "type": "integer",
+      "format": "int32",
+      "description": "Numeric ID value associated with the type of unit being displayed. Can be used for unit translation. Please refer to [Weather Service Concepts](https://aka.ms/AzureMapsWeatherConcepts) for details.",
+      "enum": [
+        0,
+        1,
+        2,
+        3,
+        4,
+        5,
+        6,
+        7,
+        8,
+        9,
+        10,
+        11,
+        12,
+        13,
+        14,
+        15,
+        16,
+        17,
+        18,
+        19,
+        20,
+        21,
+        22,
+        31
+      ],
+      "x-ms-enum": {
+        "name": "UnitType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": 0,
+            "description": "feet"
+          },
+          {
+            "value": 1,
+            "description": "inches"
+          },
+          {
+            "value": 2,
+            "description": "miles"
+          },
+          {
+            "value": 3,
+            "description": "millimeter"
+          },
+          {
+            "value": 4,
+            "description": "centimeter"
+          },
+          {
+            "value": 5,
+            "description": "meter"
+          },
+          {
+            "value": 6,
+            "description": "kilometer"
+          },
+          {
+            "value": 7,
+            "description": "kilometersPerHour"
+          },
+          {
+            "value": 8,
+            "description": "knots"
+          },
+          {
+            "value": 9,
+            "description": "milesPerHour"
+          },
+          {
+            "value": 10,
+            "description": "metersPerSecond"
+          },
+          {
+            "value": 11,
+            "description": "hectoPascals"
```

Comment:
This unknown word fails [Spell check](https://github.com/Azure/azure-rest-api-specs/pull/16148/checks?check_run_id=3751822770).  There is an instruction on how to fix that.

Reply:
[Prettier formatting](https://github.com/Azure/azure-rest-api-specs/pull/16148/checks?check_run_id=3751822760) for the Swagger should be rerun as well.